.pre-commit-config.yaml

```diff
@@ -1,4 +1,8 @@
 repos:
+  - repo: https://github.com/unifyai/lint-hook
+    rev: function-ordering
```

Comment:
We will need to change the revision before merging, I'd prefer if we pin a specific commit hash